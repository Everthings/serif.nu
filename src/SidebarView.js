import React from 'react';
import TextField from '@material-ui/core/TextField';

function SidebarView() {
  return (
    <div>
      <TextField label="Search for classes" type="search" />
    </div>
  );
}

export default SidebarView;
