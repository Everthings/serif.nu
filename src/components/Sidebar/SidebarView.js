import React from 'react';
import Search from './Search';

function SidebarView() {
  return (
    <div>
      <Search />
    </div>
  );
}

export default SidebarView;
