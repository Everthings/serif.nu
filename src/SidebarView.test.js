import React from 'react';
import renderer from 'react-test-renderer';
import SidebarView from './SidebarView';

it('renders correctly', () => {
  const tree = renderer.create(<SidebarView />).toJSON();

  expect(tree).toMatchSnapshot();
});
