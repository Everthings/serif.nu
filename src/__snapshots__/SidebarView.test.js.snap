// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="MuiFormControl-root-1"
  >
    <label
      className="MuiFormLabel-root-10 MuiInputLabel-root-5 MuiInputLabel-formControl-6 MuiInputLabel-animated-9"
      data-shrink={false}
    >
      Search for classes
    </label>
    <div
      className="MuiInput-root-17 MuiInput-formControl-18 MuiInput-underline-21"
    >
      <input
        aria-invalid={false}
        className="MuiInput-input-25 MuiInput-inputType-28 MuiInput-inputTypeSearch-29"
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="search"
      />
    </div>
  </div>
</div>
`;
